package-lock.json

```diff
@@ -380,6 +381,8 @@
           "requires": {
             "lru-cache": "^4.0.1",
             "which": "^1.2.9"
+=======
+>>>>>>> Fixed Linter Errors so not it runs.
```

Comment:
I'm not sure if I am reading this correctly, but it looks like a bunch of merge conflicts got added to the branch?
These lines will probably break the file.

Reply:
Ironically, it doesn't as long as you rerun `npm install` but it should be fixed regardless